src/app/shared/table-of-contents/table-of-contents.html

```diff
@@ -1,6 +1,6 @@
 <div *ngFor="let linkSection of _linkSections" class="docs-toc-container">
   <div class="docs-toc-heading">{{linkSection.name}}</div>
-  <nav>
+  <nav aria-label="Section Table of Contents">
```

Comment:
Instead of "Section", could we use the name of the item being documented at the time? Maybe something like "Button overview table of contents"

Reply:
We can use `linkSection.name` in the label here, but that would give the following
- "Overview content Table of Contents" for a component's overview TOC
- "Checkbox Table of Contents" for a component's API TOC

I suppose that this is better than just "Section". If we want to add something like "Checkbox overview", we would need to add another Input and pass that data in as we don't currently have that available in the TOC component.

Reply:
I think that is better, though adding another input seems pretty straightforward